test/Microsoft.AspNetCore.Mvc.FunctionalTests/Infrastructure/MvcTestFixture.cs

```diff
@@ -17,6 +17,7 @@ protected override void ConfigureWebHost(IWebHostBuilder builder)
         {
             builder
                 .UseRequestCulture<TStartup>("en-GB", "en-US")
+                .UseEnvironment("Production")
```

Comment:
Why is this required?

Reply:
Cause some of our tests will produce different results between dev/prod